target/linux/mediatek/filogic/base-files/etc/board.d/02_network

```diff
@@ -71,9 +75,12 @@ mediatek_setup_macs()
 		wan_mac=$(macaddr_add "$lan_mac" 1)
 		label_mac=$wan_mac
 		;;
+	xiaomi,mi-router-wr30u-stock|\
+	xiaomi,mi-router-wr30u-ubootmod|\
 	xiaomi,redmi-router-ax6000-stock|\
 	xiaomi,redmi-router-ax6000-ubootmod)
 		wan_mac=$(mtd_get_mac_ascii Bdata ethaddr_wan)
+		lan_mac=$wan_mac
```

Comment:
Looks wrong.

Reply:
Corrected. I recheck stock firmware, it should be removed, my understand before is wrong, thank you.

Reply:
> may be we can fix it ,wr30u mac from mtd3(Bdata), ethaddr_wan is mac ,ethaddr is 5G mac We can refer to this [padavanonly/immortalwrtARM@f76f9fc](https://github.com/padavanonly/immortalwrtARM/commit/f76f9fc1baf2e0fd502611f86d4ca23420fe1248) wan_mac=$(dd if=/dev/mtd3ro bs=1 count=256 2>/dev/null |grep "^ethaddr_wan=" | awk -F '=' '{print $2}') lan_mac=$(dd if=/dev/mtd3ro bs=1 count=256 2>/dev/null |grep "^ethaddr=" | awk -F '=' '{print $2}')



> may be we can fix it ,wr30u mac from mtd3(Bdata), ethaddr_wan is mac ,ethaddr is 5G mac We can refer to this [padavanonly/immortalwrtARM@f76f9fc](https://github.com/padavanonly/immortalwrtARM/commit/f76f9fc1baf2e0fd502611f86d4ca23420fe1248) wan_mac=$(dd if=/dev/mtd3ro bs=1 count=256 2>/dev/null |grep "^ethaddr_wan=" | awk -F '=' '{print $2}') lan_mac=$(dd if=/dev/mtd3ro bs=1 count=256 2>/dev/null |grep "^ethaddr=" | awk -F '=' '{print $2}')

It is unneeded. Everything works fine. WIFI, Lan and Wan MACs are all correct.